xbmc/cores/VideoPlayer/Interface/TimingConstants.h

```diff
@@ -8,10 +8,15 @@
 
 #pragma once
 
+#include <stdint.h>
```

Comment:
remove this header

Reply:
Compiler error: `error: ‘int64_t’ does not name a type`
I guess I should include something to make this work.

Reply:
ah ok then in this case use c++ header `<cstdint>`

Reply:
Okay, thanks.